src/Models/AzureFunctions.ts

```diff
@@ -85,6 +85,33 @@ export class AzureFunctions implements Component, Provisionable, Deployable {
     return this.componentType;
   }
 
+  static async isAvailable(): Promise<boolean> {
+    if (!vscode.extensions.getExtension(
+            'ms-azuretools.vscode-azurefunctions')) {
+      const choice = await vscode.window.showInformationMessage(
+          'Azure Functions extension has been disabled or not installed. Please enable Azure Functions extension or install it from marketplace, and try again.',

```

Comment:
Is it better to use:    Azure Functions extension is required for the current project. Do you want to install it from marketplace?